### PR TITLE
Update Rabbitmq version to 3.12

### PR DESCRIPTION
**Pull request by ruaridhg**
Updated rabbitmq version to 3.12-management in tests for modules:
- pixl imaging
- cli
- core
- system

Needed to `export DOCKER_DEFAULT_PLATFORM=linux/arm64` for it to work on Mac M1

### CODE REVIEW

File: pixl_imaging/tests/docker-compose.yml [hunk 1]
```diff
@@ -54,6 +54,7 @@ services:
       - "9001:8000"
 
   vna-qr:
+    platform: linux/amd64
```
Comment (stefpiatek) [inline pixl_imaging/tests/docker-compose.yml:57]: are these all needed?

**Review by stefpiatek — APPROVED** [review]
Would be good to check that we need to specify the platform locally

File: pixl_imaging/tests/docker-compose.yml (hunk 1 above)
Comment (ruaridhg) [reply pixl_imaging/tests/docker-compose.yml:57]: Having looked into it the answer appears to be yes. Also need to set `DOCKER_DEFAULT_PLATFORM=linux/arm64` for it to work locally which I'm not sure why.

Error when no platform is specified:
`Error response from daemon: image with reference osimis/orthanc:22.9.0-full-stable was found but does not match the specified platform: wanted linux/arm64, actual: linux/amd64 `

Error if I set `DOCKER_DEFAULT_PLATFORM=linux/amd64` (both with and without platform specified):
`Error response from daemon: image with reference rabbitmq:3.12.9-management was found but does not match the specified platform: wanted linux/amd64, actual: linux/arm64`

Comment (ruaridhg) [reply pixl_imaging/tests/docker-compose.yml:57]: Same pattern of errors for the system tests

Comment (stefpiatek) [reply pixl_imaging/tests/docker-compose.yml:57]: I think you'll need to down all of your containers and then do `docker system prune --all` as its downloaded that version on arm and doesn't seem to want to redownload it

Comment (milanmlft) [reply pixl_imaging/tests/docker-compose.yml:57]: Yeah, I don't think all that is necessary, I'm able to run these containers fine on my M2 (would surprise me if there would be a difference between M1-M2 in this case).
Make sure you have Rosetta enabled as well in Docker settings.
Screenshot 2024-02-09 at 09 52 17

**Comment by stefpiatek** [comment]
Happy for you to emrge once you've resovled the conflicts

**Review by stefpiatek — APPROVED** [review]

**Pull request merged**